src/analyzer/protocol/syslog/syslog.spicy

```diff
@@ -15,9 +15,9 @@ public type Message = unit {
     on %done {
         spicy::accept_input();
 
-        if ( self?.prio ) {
+        if (self?.prio) {
             self.severity = (self.prio.value & 0x07);
-            self.facility = (self.prio.value & 0x03f8) >> 3;
+            self.facility = (self.prio.value & 0x03f8)>>3;
```

Comment:
Seems worth a fix :-)

Reply:
Shift operators are currently unhandled and adding support for them would be great. The rule that would need to be expanded is https://github.com/bbannier/spicy-format/blob/8843f49a2291fb8f4d2f82724b4d76e1752659fd/src/query.scm#L196-L214

Reply:
https://github.com/bbannier/spicy-format/pull/127 has landed with spicy-format-0.15.0 so this should format like expected if you bump the version in `.pre-commit-config.yaml`.